src/components/header/header.scss

```
@@ -0,0 +1,116 @@
+@import '/src/styles/variables.scss';
+
+.Header {
+  @include elementPosition(flex, column, center);
+  align-items: center;
+
+  .topMenuListWrap {
+    @include elementPosition(flex, row, center);
+    width: 100%;
+    background: #fafafa;
+
+    .topMenuList {
+      @include elementPosition(flex, row, flex-end);
+      align-items: center;
+      height: 40px;
+      width: 1140px;
+
+      .topMenuWrap {
+        margin-left: 20px;
+
+        .topMenu {
+          font-size: 12px;
+        }
+      }
+
+      .topMenuWrap:first-child {
+        margin-left: 0;
+      }
+    }
+  }
+}
+
+.headerWrap {
+  @include elementPosition(flex, row, space-between);
+  width: 1140px;
+  height: 85px;
+  position: relative;
+  align-items: center;
+
+  .navigationBtn {
+    width: 27px;
+    height: 25px;
+    border: none;
+    cursor: pointer;
+
+    .navigationBtnIcon {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+
+  .headerLogo {
+    position: absolute;
+    width: 167px;
+    height: 95px;
+    top: -25px;
+    left: 85px;
+  }
+
+  .searchInput {
+    width: 392px;
+    height: 28px;
+    font-size: $fontMedium;
+    background-color: transparent;
+    border: none;
+    margin-right: 35px;
+  }
+
+  .searchAndIconWrap {
+    display: flex;
+  }
+
+  .iconListWrap {
+    display: flex;
+    align-items: center;
+
+    .iconWrap {
+      margin-left: 32px;
+      cursor: pointer;
+      background: #fff;
+      border: none;
+
+      .headerIcon {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+      }
+    }
+
+    .iconWrap:nth-child(1) {
+      width: 32px;
+      height: 32px;
+    }
```

```suggestion
    .iconWrap {
      &:nth-child(1){
        width: 32px;
        height: 32px;
      }  
    }
```
& 연산자 잘 활용하면 iconWrap안에 다 네스팅해서 간결하게 적을 수 있습니다